src/layouts/InterfaceLayout/InterfaceLayout.vue

```diff
@@ -437,34 +437,41 @@ export default {
       this.receivedTokens = false;
       let tokens = [];
       if (
-        this.network.type.chainID === 1 ||
-        (this.network.type.chainID === 3 &&
-          !this.network.url.includes('infura'))
+        (this.network.type.chainID === 1 || this.network.type.chainID === 3) &&
+        !this.network.url.includes('infura')
       ) {
         const tb = new TokenBalance(this.web3.currentProvider);
         try {
-          tokens = await tb.getBalance(
-            this.account.address,
-            true,
-            true,
-            true,
-            0,
-            { gas: '0x11e1a300' }
-          );
+          tokens = await tb.getBalance(this.account.address, true, true, true, {
+            gas: '0x11e1a300'
+          });
           tokens = tokens.map(token => {
             token.address = token.addr;
             delete token.addr;
             return token;
           });
+
+          const filteredNetwork = this.network.type.tokens.filter(token => {
+            const found = tokens.find(item => {
+              return (
+                this.web3.utils.toChecksumAddress(item.address) ===
+                this.web3.utils.toChecksumAddress(token.address)
+              );
+            });
+
+            if (!found) return token;
+          });
+
+          tokens = tokens.concat(filteredNetwork);
         } catch (e) {
           tokens = this.network.type.tokens.map(token => {
-            token.balance = 'Load';
+            token.balance = 0;
```

Comment:
Shouldn't this still be Load? On network error they should be able to manually Load

Reply:
right. I'll change it